SPECS-EXTENDED/aopalliance/aopalliance.spec

```diff
@@ -87,6 +87,9 @@ cp -pr build/javadoc/* %{buildroot}%{_javadocdir}/%{name}/
 %{_javadocdir}/%{name}
 
 %changelog
+* Tue 08 2024 Durga Jagadeesh Palli <v-dpalli@microsoft.com> - 1.0-4
```

Comment:
We have missed to update the `Release` tag.

Reply:
Hi @mfrw, I have updated the Release tag.